proxy-manager/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/base-nodejs:0.2.2
+ARG BUILD_FROM=ghcr.io/hassio-addons/base-nodejs:0.2.5
```

Comment:
This isn't related to this PR / shouldn't be in this PR.

Focus pull request on a single thing at the time. Upgrading the base image a task of upgrading npm.